crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -333,13 +327,28 @@ async fn conclude_handler(
         user,
     )
     .await?;
+
+    if let (true, Some(experiments_webhook_config)) = (
+        tenant_config.experiments_webhook_config.enabled,
+        tenant_config.experiments_webhook_config.configuration,
```

Comment:
I thought we added enabled to avoid parsing configuration?

Reply:
But this check would be needed right?